### ISSUE TITLE
Cursor feeling heavy / delayed when I am moving it

### ISSUE THREAD

**Issue by BetaDv** [LibreSprite/LibreSprite#310]
##### LibreSprite version
Libresprite 1.2-973d0e3d-SDL

##### Describe the bug
Whenever I move my mouse, the cursor feels heavy or is delayed sometimes. Unsure why that happens so I opted out to the Bug Report.

##### To Reproduce
well idk I just booted up the program

##### Expected behaviour
mouse to feel normal like it usually does and quick

##### OS/Hardware
- Operative System: Windows 10 20H2
- CPU: Intel Core I5-8265U, 1.6 GHz
- GPU: Intel UHD Graphics 620 and NVIDIA GeForce MX230 (I have 2)
UNSURE IF IT NEEDED THIS FOR MY ISSUE

**Comment by Zughy** [comment]
Hello, thank you for the report. Have you tried on the stable Windows version? You can find it here https://github.com/LibreSprite/LibreSprite/releases/tag/v1.0

**Comment by BetaDv** [comment]
but that's what I installed

**Comment by Zughy** [comment]
..right, we haven't updated the name. Does it also happen with the native mouse cursor enabled? Edit -> Preferences -> Cursors -> Use native mouse cursor

**Comment by BetaDv** [comment]
yea, I searched a bit in the issues tab and I found that, still feels the same

**Comment by veganaize** [comment]
Under `Edit` -> `Preferences` -> `General`
Is "**Screen Scaling**" at **200%** and "**UI Elements Scaling**" at **100%**?

**Comment by pegvin** [comment]
This is what @BetaDv means, basically he's moving the mouse and on screen it's moving but a little delayed & that is what creates the "heavy" effect. my best guess is it occurs because the virtual mouse positions are updated "late" because it's done on the CPU side and your CPU doesn't seem that powerful.

**Comment by veganaize** [comment]
I seem to get a similar effect if my scaling settings aren't at the defaults.

**Comment by pegvin** [comment]
scaling might effect it because it has to do more work than usual.

**Comment by veganaize** [comment]
Exactly.  I was trying to see if @BetaDv had their scaling at the defaults.

**Comment by DMClVG** [comment]
I've have noticed the same issue. After disabling "Cursors/Brush Preview", my cursor moves without visible delay when it's not drawing on the canvas. 

BTW I'm using a stylus

**Comment by aladvs** [comment]
I've noticed this too. Seems to be less apparent at smaller resolutions. Very apparent at 1440p but less so at 1080p.
